uiowa/uiowa
# Remove Schemadotorg

Relates to https://github.com/uiowa/uiowa/issues/6847




# How to test



On `main`:

```
ddev drush @default.local sql-drop && ddev drush @default.local cr && ddev blt di --site=default
```

On this branch, repeat and see there is no issue with person entity_view_display configuration.

You can also test events on a site to double-check that removing the schemadotorg function from the sitenow_events module did not cause any issues.
